src/System.Management.Automation/namespaces/FileSystemProvider.cs

```diff
@@ -8105,15 +8105,18 @@ internal unsafe struct WIN32_FIND_DATA
         /// </summary>
         /// <param name="instance">The object of FileInfo or DirectoryInfo type.</param>
         /// <returns>The target of the reparse point.</returns>
+        [Obsolete("This method is now obsolete. Please use the .NET API 'FileSystemInfo.LinkTarget'", error: true)]
```

Comment:
```suggestion
        [Obsolete("This method is now obsolete. Please use the .NET 6.0+ API 'FileSystemInfo.LinkTarget'", error: true)]
```

Reply:
I don't think this is needed. A user will run in this obsolete error only if they are compile code against 7.2+ version of PowerShell NuGet packages, and then they are using .NET 6+ for sure.

Reply:
It is more for code readers :-)